js/scripts.js

```diff
@@ -0,0 +1,13 @@
+var text = "Papugi koszą trawę, unosząc się nad ziemią 15 centymetrów, na czerwonych dywanach.";
+var animal = prompt("nazwa nowych zwierzątek:");
+
+console.log("Text oryginalny: " + text); // pobieramy nazwe nowego animalsa
+
+var tablica = text.split(' '); // rozwalamy  text na tablice
+tablica[0] = animal; // pierwszy element tablicy = pierwszemu wyrazowi podmieniamy na nowe zwierzatko
+text=tablica.join(" "); // sklajamy tablice w text ponownie
```

Comment:
jak to jest, że dwie linijki robisz spacje przy przypisaniu wartości, a linijkę niżej już nie? To nie jest kwestia niewiedzy